pkg/ingester/metrics.go

```diff
@@ -60,7 +60,8 @@ type ingesterMetrics struct {
 	// Open all existing TSDBs metrics
 	openExistingTSDB prometheus.Counter
 
-	discarded *discardedMetrics
+	discarded         *discardedMetrics
+	discardedInstance *prometheus.CounterVec
```

Comment:
Should we add this to `discardedMetrics`? It's not clear to me what belongs in there and what goes here (we've got the discarded metadata metrics below, but everything else related to discarding data seems to be on `discardedMetrics`)

Reply:
Changed to `rejected` per Peter's comments so they're now separate and named differently.